.github/ISSUE_TEMPLATE/Style.yml

```diff
@@ -0,0 +1,31 @@
+name: 👯‍♂️ Style Changing Request
+description: Suggest a style designs
+title: '[style]: '
```

Comment:
The title format across templates are all different. It is better to uniform the format of title. For example, `[style] <title>`, `[feature] <title>` and `[bug] <title>`.